biocypher_metta/adapters/enhancer_atlas_adapter.py

```diff
@@ -0,0 +1,99 @@
+import gzip
+import os
+import pickle
+from biocypher_metta.adapters import Adapter
+from biocypher_metta.adapters.helpers import build_regulatory_region_id, check_genomic_location
+
+# Example enhancer atlas input file:
+
+# enhancer_file
+# chr1	875310	876520	11.4349429027718
+# chr1	876690	877510	10.6950350518485
+# chr1	902000	902080	10.429771917918
+# chr1	902180	902820	10.3902750575485
+
+# enhancer_gene_file
+# chr1:874840-876520_ENSG00000225880$LINC00115$chr1$762902$-	1.104330
```

Comment:
Can you add some text describing the last column? What do the numbers represent?